authentikos/go.mod

```diff
@@ -0,0 +1,13 @@
+module istio.io/test-infra/authentikos
```

Comment:
Why do we need/want this to be its own module?

Reply:
I added this mainly to optimize the docker build, so that I can both build in the container and so the image only pulls in the necessary dependencies specifically for this code.